src/main/java/Task.java

```diff
@@ -0,0 +1,65 @@
+public class Task {
+    /** name of task input by user. */
+    public String description;
+    /** True means task complete, otherwise if false. */
+    public boolean isDone;
+    /** Type of task (eg. todo, Deadline, event) stored as string */
+    public String type;
+
+    /**
+     * Constructor for new object of Task class.
+     *
+     * @param description Name of task.
+     */
+    public Task(String description, String taskType) {
+        this.description = description;
+        this.isDone = false;
+        this.type = taskType;
+    }
+
+    /**
+     * Setter to change status of task to done.
```

Comment:
Perhaps change this comment to `Sets task as done`?

As per the coding standard, for Javadoc comments, "the first sentence should start in the form Returns ..., Sends ..., Adds ... (not Return or Returnning etc.)" (https://se-education.org/guides/conventions/java/basic.html#comments)

Reply:
OK, Didn't know about this.